hwi/aladin_crawling.py

```
@@ -0,0 +1,99 @@
+    print(title, link, price, star_score)
+
+
+# ### 여러 페이지 크롤링
+
+# In[111]:
+
+
+data = []
+for page_num in range(1, 4):  # 1~3페이지 크롤링
+    url = f"https://www.aladin.co.kr/shop/common/wbest.aspx?BestType=Bestseller&BranchType=1&CID=0&page={page_num}&cnt=1000&SortOrder=1"
+    response = requests.get(url)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)